From e887b657331cbfd50aff24c4b882011fc51f3c35 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Wed, 22 Aug 2018 10:12:31 +0200
Subject: [PATCH 9/9] miamimp: Load FPGA bitstream at boot

Workaround since FPGA firmware loading is broken in Xilinx kernel 4.14
---
 include/configs/topic_miamimp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/topic_miamimp.h b/include/configs/topic_miamimp.h
index f749c8b..82b6ef8 100644
--- a/include/configs/topic_miamimp.h
+++ b/include/configs/topic_miamimp.h
@@ -66,6 +66,7 @@
 	"sdboot=mmc dev $sdbootdev && mmcinfo && " \
 		"if load mmc $sdbootdev:$partid 0x3F00000 autorun.scr; then source 0x3F00000; else; run uenvboot; fi; "\
 		"run sdroot$sdbootdev; " \
+		"if load mmc $sdbootdev:2 0x100000 /lib/firmware/fpga.bin; then fpga load 0 0x100000 $filesize; fi; "\
 		"load mmc $sdbootdev:$partid $fdt_addr system.dtb && " \
 		"load mmc $sdbootdev:$partid $kernel_addr Image && " \
 		"booti $kernel_addr - $fdt_addr\0" \
-- 
1.9.1

